### PR TITLE
Bunch of updates (Sorry idk why this didn't do different pull requests)

### PR DESCRIPTION
**Pull request by ThatFridgeFella**
Rewrote the asset modification rules to include categories for each type of modification, and also made the rules a bit more clear and consistent with how we deal with asset modifications normally. (Please GOD let me have done this right this time)

### CODE REVIEW

**Comment by ThatFridgeFella** [comment]
Ok somehow this turned into way bigger of an update and I still suck at github, but im still adding stuff, give me a second.

**Comment by ThatFridgeFella** [comment]
ok I think everything is done, sorry again for the mega pr, I know what I did wrong and I will do better next time.

File: content/02_general-rules/04_asset-modifications.md [hunk 1]
```diff
@@ -1,10 +1,63 @@
...
 
-Texture modifications are allowed, with some caveats. The textures of portal surfaces
-must not be modified in any way. No textures can be modified to provide any visual
-assistance, such as help with a lineup. No texture may be modified to be more
-translucent than it was, with the exception of the player's viewmodel.
-
-Modifying models is not allowed, with the exception of the portal gun (including both
-the viewmodel and the in-world model). Modifying audio files, captions or any other
-asset is not allowed.
+The asset modification rules will be split into their respective categories that are
```
Comment (mlugg) [inline content/02_general-rules/04_asset-modifications.md:3]: Don't describe how the rules document is laid out here, people can already see that: just say something like "We have strict rules regarding which assets are allowed to be changed from their defaults during runs, which (unless otherwise specified) are shared across all games and categories. These rules vary depending on the kind of asset being modified." and go into the sections

File: content/02_general-rules/04_asset-modifications.md [hunk 2]
```diff
@@ -1,10 +1,63 @@
...
+a “General Rules” category that applies to any kind of asset modification.
+
+#### General Rules
+
+Any asset modifications are subject to being grounds for the rejection or removal of 
+any run on speedrun.com or the cm boards.
+
+When assets are required to be submitted alongside a run, these assets must adhere to 
+speedrun.com TOS and any other applicable TOS for wherever your proof of the run is 
+being stored. Including video submissions.
```
Comment (mlugg) [inline content/02_general-rules/04_asset-modifications.md:14]: Let's not tie the language into SRDC more than we need to - also, the grammar here is a little questionable. Suggest alternative: "the TOS of all platforms involved in your submission, including both the site the run is submitted to and any sites linked from the submission."

File: content/02_general-rules/04_asset-modifications.md [hunk 3]
```diff
@@ -1,10 +1,63 @@
...
+
+Editing .dll files that are present in the files to modify an asset is not allowed.
```
Comment (mlugg) [inline content/02_general-rules/04_asset-modifications.md:16]: Suggest edit: "Editing binary files (including .DLL, .EXE, .SO, A.OUT, .DYLIB, or .APP) files which are present in the game or Steam directory..."

Although, shouldn't we really just state somewhere that this is blanket disallowed? I thought we already did, but I could be wrong.

File: content/02_general-rules/04_asset-modifications.md [hunk 4]
```diff
@@ -1,10 +1,63 @@
...
+
+If you have any questions about whether something is allowed or not, please ask.
```
Comment (mlugg) [inline content/02_general-rules/04_asset-modifications.md:18]: This line feels a bit redundant, but let's at least clarify *who* to ask (category moderators).

File: content/02_general-rules/04_asset-modifications.md [hunk 5]
```diff
@@ -1,10 +1,63 @@
...
+
+#### Textures
+
+Portalable surfaces cannot be modified for any reason.
+
+No textures can be modified to provide any visual assistance, such as help with a lineup.
+
+Textures can not be more transparent than their original texture, this tends to be seen 
+most in light bridge modifications. The only exception to this rule is the player’s
```
Comment (mlugg) [inline content/02_general-rules/04_asset-modifications.md:27]: Suggest reword: "...original texture (be particularly careful of this when modifying the light bridge texture)."

File: content/02_general-rules/04_asset-modifications.md [hunk 6]
```diff
@@ -1,10 +1,63 @@
...
+
+No textures can be modified to provide any visual assistance, such as help with a lineup.
+
+Textures can not be more transparent than their original texture, this tends to be seen 
+most in light bridge modifications. The only exception to this rule is the player’s 
+viewmodel for both the portal gun and PotatOS.
+
+#### Models
+
+Modifying models of assets is not allowed, except for both the world model and the
```
Comment (mlugg) [inline content/02_general-rules/04_asset-modifications.md:32]: "models of assets" is meaningless: just "models" is fine, or perhaps "visual or physics models" to clarify that you shouldn't mess with either

File: content/02_general-rules/04_asset-modifications.md [hunk 7]
```diff
@@ -1,10 +1,63 @@
...
+
+#### Models
+
+Modifying models of assets is not allowed, except for both the world model and the 
+viewmodel of the portal gun. The world model of the portal gun cannot be used to gain an 
+advantage for any reason.
+
+#### Particles
+
+Particles can be changed only if it doesn’t require editing any of the game’s .dll files.
```
Comment (mlugg) [inline content/02_general-rules/04_asset-modifications.md:38]: Again, feels redundant. In this case, maybe reword to focus on what *is* allowed: "It is permitted to modify particles by modifying game data files (i.e. not binaries)."

File: content/02_general-rules/04_asset-modifications.md [hunk 8]
```diff
@@ -1,10 +1,63 @@
...
+
+Particle changes cannot be used for an advantage, nor can they be entirely removed.
+
+#### Resource
+
+Resource files can be changed, but not to gain an advantage or to outright remove them.
+
+Any resource file that changes how the menu is structured, or how they function is not 
+allowed to be active during singleplayer SLA runs.
```
Comment (mlugg) [inline content/02_general-rules/04_asset-modifications.md:47]: Category-specific rules should go in the relevant category. Remove this section, and in the any% extra rules section write something like "The structure and functionality of game menus may not be modified in any capacity, by modifying resource files or otherwise."

File: content/02_general-rules/04_asset-modifications.md [hunk 9]
```diff
@@ -1,10 +1,63 @@
...
+
+#### Scripts
+
+Script files cannot be changed, regardless of the reasoning.
+
+#### Sound
+
+Sound files can be changed, but not to gain an advantage.
+
+Voice lines that are integral to the run cannot be changed, this includes but is not
```
Comment (mlugg) [inline content/02_general-rules/04_asset-modifications.md:61]: Grammar: these are separate sentences

File: content/05_games/00_portal-2.cats [hunk 10]
```diff
@@ -8,6 +8,7 @@ Name,Objective,OOB,SLA,"Pause Abuse","Demo Requirement",Leaderboard,Notes,Modera
...
-"Portal Placement Never Fail","Complete the specified campaign according to the standard ruleset for the underlying category.",Yes,-,-,"All runs",https://www.speedrun.com/portal_2_category_extensions?h=Portal_Placement_Never_Fail-SP&x=wkpo8182-38dw2v1n.8102832q,"You may use `sv_portal_placement_never_fail 1` (which requires `sv_cheats 1`). All other rules come from the underlying category (Inbounds No SLA, AMC, Co-op any%, or Solo Co-op).","Betsruner, BiSaXa, Fridge, phunkpai"
+"Portal Placement Never Fail","Complete the specified campaign according to the standard ruleset for the underlying category.",Yes,No,Yes,"All runs",https://www.speedrun.com/portal_2_category_extensions?h=Portal_Placement_Never_Fail-SP&x=wkpo8182-38dw2v1n.8102832q,"You may use `sv_portal_placement_never_fail 1` (which requires `sv_cheats 1`). All other rules come from the underlying category (Inbounds No SLA, AMC, Co-op any%, or Solo Co-op).","Betsruner, BiSaXa, Fridge, phunkpai"
```
Comment (mlugg) [inline content/05_games/00_portal-2.cats:11]: These fields are intentionally blanked because these rules are inherited from the underlying category: e.g. you can't pause abuse in coop.

File: content/05_games/00_portal-2.cats [hunk 11]
```diff
@@ -8,6 +8,7 @@ Name,Objective,OOB,SLA,"Pause Abuse","Demo Requirement",Leaderboard,Notes,Modera
...
 "Singleplayer 100 Portals","Complete the entire singleplayer campaign within 100 portals.",No,No,Yes,200th,https://www.speedrun.com/portal_2_category_extensions?h=Least_Portals_Challenge_Runs-SP_100_Portals&x=9d83ywl2-ylqqryzl.z1903g81,"You must place no more than 100 portals during the run. `sar_lphud` can be used to track portal shots.","Betsruner, BiSaXa, Fridge, phunkpai"
 "Co-op 69 Portals","Complete all courses (1-6) of the cooperative campaign within 69 portals.",Yes,Yes,No,200th,https://www.speedrun.com/portal_2_category_extensions?h=Least_Portals_Challenge_Runs-Co-op_69_Portals&x=9d83ywl2-ylqqryzl.814rjevq,"You must place no more than 69 portals during the run. `sar_lphud` can be used to track portal shots.","Betsruner, BiSaXa, Fridge, phunkpai"
+"No Major Glitches","Complete the entire singleplayer campaign.",No,No,No,5th,https://www.speedrun.com/portal_2_category_extensions?h=No_Major_Glitches&x=zdn458xk,"You must complete the game without the use of a major glitch, the list of major glitches can be found here: https://docs.google.com/document/d/10F62Cgars4I-kZ-alOjptZ64ozSMIJ4_j1r26kTGXHc/edit?usp=sharing","Betsruner, BiSaXa, Fridge, phunkpai"
```
Comment (mlugg) [inline content/05_games/00_portal-2.cats:14]: grammar: "...without the use of a major glitch. A list of major glitches can be found at https://..."

File: content/05_games/01_speedrun-mod.cats [hunk 12]
```diff
@@ -3,3 +3,5 @@ Normal Mode,"Complete the game in Normal Mode.",No,No,Yes,200th,https://www.spee
 Celeste Mode,"Complete the game in Celeste Mode.",No,No,Yes,10th,https://www.speedrun.com/p2sm?h=Celeste_Mode-Any&x=n2yomp1d-38dj7ye8.81w34o5l,,"Betsruner, BiSaXa, Fridge"
 Fog%,"Complete the game in Fog%.",No,No,Yes,200th,https://www.speedrun.com/p2sm?h=Fog&x=9kvoq5e2,"`mat_ambient_light_r/g/b` and `mat_fullbright` are banned. On top of the demo requirement, video evidence is required for all runs.","Betsruner, BiSaXa, Fridge"
 Reverse%,"Complete the game in Reverse Mod.",No,No,Yes,200th,https://www.speedrun.com/p2sm?h=Reverse&x=zd35ezrk,,"Betsruner, BiSaXa, Fridge"
+Floor Is Lava,"Complete the game in the Floor is Lava Mode",No,No,Yes,Any Runs,https://www.speedrun.com/p2sm?h=Floor_is_Lava&x=02qwr772,,"Betsruner, BiSaXa, Fridge"
+The Ultimate Speedrun,"Complete the game using extra commands to emulate CS:S styled movement",No,No,Yes,Any Runs,https://www.speedrun.com/portal_2_category_extensions?h=The_Ultimate_Speedrun&x=zdn7g6nk,"Runs must be done with `sv_cheats 1`, `sv_airaccelerate 1000`, `sar_aircontrol 2`, `sar_autojump 1`, `portalgun_fire_delay 0`. Runs can also use `portalgun_held_button_fire_fire_delay 0 or 0.50`.,"Betsruner, BiSaXa, Fridge"
```
Comment (mlugg) [inline content/05_games/01_speedrun-mod.cats:7]: Consistency: "All runs" (capitalization)

File: content/05_games/03_reloaded.cats [hunk 13]
```diff
@@ -1,3 +1,4 @@
 Name,Objective,OOB,SLA,"Pause Abuse","Demo Requirement",Leaderboard,Notes,Moderators
 "No SLA","Complete the entire campaign with either ending.",No,No,Yes,25th,https://www.speedrun.com/p2reloaded?h=No_SLA&x=02q93w7k,`developer 1` is banned.,"Betsruner, BiSaXa, Fridge"
 "SLA","Complete the entire campaign with either ending.",Yes,Yes,Yes,25th,https://www.speedrun.com/p2reloaded?h=SLA&x=824lgeed,`developer 1` is banned.,"Betsruner, BiSaXa, Fridge"
+"Coop","Complete the entire coop campaign.",Yes,Yes,No,Any Runs,https://www.speedrun.com/p2reloaded,`developer 1` is banned.,"Betsruner, BiSaXa, Fridge"
```
Comment (mlugg) [inline content/05_games/03_reloaded.cats:4]: Consistency: "All runs"

File: content/05_games/05_sourcemods.cats [hunk 14]
```diff
@@ -0,0 +1,19 @@
...
+Mind Escape - Inbounds SLA,"Complete the single player campaign",No,Yes,Yes,Any Runs,https://www.speedrun.com/mind_escape?h=Inbounds_SLA&x=xd14z57d,"Timing ends on interacting the door to the van, starting the final cutscene.","Betsruner, Infrabread, KiloOnline, Fridge"
+Mind Escape - Any%,"Complete the single player campaign",Yes,Yes,Yes,Any Runs,https://www.speedrun.com/mind_escape?h=Any&x=zd3ozq8d,"Timing ends on interacting the door to the van, starting the final cutscene.","Betsruner, Infrabread, KiloOnline, Fridge"
+Portal: Unity Reboot - Inbounds No SLA,"Complete the single player campaign",No,No,Yes,Any Runs,https://www.speedrun.com/portal_unity_reboot?h=Inbounds_No_SLA-Legacy&x=xk9pn04d-wl39wwvl.21dpkp51,"Timing ends on the fadeout of the final level","Infrabread, Msushi, Fridge"
+Portal: Unity Reboot - Any%,"Complete the single player campaign",Yes,Yes,Yes,Any Runs,https://www.speedrun.com/portal_unity_reboot?h=Any-Legacy&x=z27mog02-wl39wwvl.21dpkp51,"Timing ends on the fadeout of the final level","Infrabread, Msushi, Fridge"
```
Comment (mlugg) [inline content/05_games/05_sourcemods.cats:19]: Consistency: "All runs"

File: content/05_games/05_sourcemods.md [hunk 15]
```diff
@@ -0,0 +1,16 @@
...
+#### Timing
+
+- Timing begins when you have gained control of the player.
+- The ending timing will be listed with the category.
+- Exceptions will be listed if there are any.
+
+#### General Rules
+
+`Developer 1` cannot be used to make the player move early, any runs that used
+developer 1 will be grandfathered in, and 1.5 seconds will be added per map where it was deemed to be an advantage.
```
Comment (mlugg) [inline content/05_games/05_sourcemods.md:12]: The rules page lists the rules *in their current form*, and is not concerned with the details of historical rule changes. Also, don't capitalize commands.
"`developer 1` cannot be used to make the player move early."

**Review by mlugg — CHANGES_REQUESTED** [review]

**Review by ThatFridgeFella — COMMENTED** [review]
done
